src/frontend/get-jira-configuration.ts

```diff
@@ -38,22 +35,32 @@ export async function getInstallation(client, subscription) {
 			subscription.repoSyncState?.numberOfSyncedRepos || 0;
 		response.data.jiraHost = subscription.jiraHost;
 
-		response.data.syncStatus === "FAILED" && sendFailedStatusMetrics(id);
+		response.data.syncStatus === "FAILED" && sendFailedStatusMetrics(id, req);
 
 		return response.data;
 	} catch (err) {
+		req.log.error(
```

Comment:
if req is optional this one might fail if it is undefined, `req?.log` should do the thing

Reply:
It will be better to jus pass logger instead of the whole request, hence these methods don't use any other stuff from the request.